web/src/components/app-survey/SurveyCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslation } from 'react-i18next';
+
+import FeedbackCard, { SurveyResponseProps } from './FeedbackCard';
+
+async function postSurveyResponse({
+  feedbackScore,
+  inputText,
+  reference: surveyReference,
+  location = 'Unknown',
+}: SurveyResponseProps) {
+  try {
+    const response = await fetch(
+      `https://hooks.zapier.com/hooks/catch/14671709/3vzj7zn/`,
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          score: feedbackScore,
+          feedback: inputText,
+          reference: surveyReference,
+          location,
+        }),
+      }
+    );
+
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+
+    const result = await response.json();
+    if (result.result === 'success') {
+      console.log('Feedback submitted successfully.');
+    } else {
+      console.error('Error submitting feedback:', result);
+    }
+  } catch (error) {
+    console.error('Error submitting user feedback:', error);
+  }
+}
+
+export default function SurveyCard() {
+  const { t } = useTranslation();
+  return (
+    <div className="invisible  z-20  sm:visible ">
+      <FeedbackCard
+        surveyReference="Map Survey"
+        postSurveyResponse={postSurveyResponse}
+        primaryQuestion={t('feedback-card.primary-question')}
+        secondaryQuestionHigh={t('feedback-card.secondary-question-high')}
+        secondaryQuestionLow={t('feedback-card.secondary-question-low')}
```

Comment:
Down the line, I'm unsure if it's worth using translations for short-lived features like this one, but I guess it can't hurt to have the option for adding multiple translations available ¯\\\_(ツ)\_/¯

Reply:
Yeah I agree, perhaps we use the callerLocation to target certain regions for feedback in the future but for now I think it's not really necessary.